com.unity.netcode.adapter.utp/Runtime/UnityTransport.cs

```diff
@@ -724,6 +727,30 @@ private void ExtractNetworkMetricsFromPipeline(NetworkPipeline pipeline, Network
                 networkMetricsContext->PacketReceivedCount = 0;
             }
         }
+
+        private void ExtractRtt(NetworkConnection networkConnection, ulong transportClientId)
+        {
+            if (NetworkManager.Singleton.IsServer)
```

Comment:
Just spot checking, are you adding new usage of `NetworkManager.Singleton` or is this the existing pattern in the transport adapter?